### DISCUSSION TITLE
why echo $shell doesnt work

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [end-4/dots-hyprland#550] [Q&A] [answered]
why does this not work and how do i fix it
![ss](https://github.com/end-4/dots-hyprland/assets/145592032/53c5b697-3ba4-45d4-84d2-5382711ebc6b)

**Comment by end-4** [comment, marked as answer]
it's $SHELL not $shell i think

**Comment by AKA-Akhil** [comment]
i feel so dumb but thank you :|
